docs/pages/index.rst

```diff
@@ -9,26 +9,74 @@ When rendering a scene from the Matterport3D dataset, *Habitat-Sim* achieves
 over 10,000 FPS multi-process on a single GPU!
 
-`Tutorials`_
-============
+The Habitat platform includes *Habitat-Sim* and `Habitat-Lab <http://aihabitat.org/docs/habitat-lab/>`_. To learn how these fit together, see our `ECCV 2020 tutorial series <https://aihabitat.org/tutorial/2020/>`_.
 
-.. TODO: this is waiting on m.css to propagate page titles to links
+Tutorials
+=========
 
--   :ref:`New Actions <std:doc:new-actions>`
--   :ref:`Notebooks <std:doc:notebooks>`
--   :ref:`Creating a stereo agent <std:doc:stereo-agent>`
--   :ref:`Working with light setups <std:doc:lighting-setups>`
--   :ref:`Extracting Images <std:doc:image-extractor>`
--   :ref:`Interactive Rigid Objects <std:doc:rigid-object-tutorial>`
+.. class:: m-table m-fullwidth
 
-Checkout our `ECCV 2020 tutorial series`_ for quick-start code and video overviews of many core features!
+=================================================== ======================================================================================== ======================
+Basics for Navigation                               `Video <https://youtu.be/kunFMRJAu2U?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/master/examples/tutorials/colabs/ECCV_2020_Navigation.ipynb>`__
 
-.. _ECCV 2020 tutorial series: https://aihabitat.org/tutorial/2020/
+Interaction                                         `Video <https://youtu.be/6eh0PBesIgw?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/master/examples/tutorials/colabs/ECCV_2020_Interactivity.ipynb>`__
 
-.. block-info:: C++ API documentation
+Advanced Topics                                     `Video <https://youtu.be/w_kDq6UOKos?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/master/examples/tutorials/colabs/ECCV_2020_Advanced_Features.ipynb>`__
 
-    Habitat-sim is designed to be used primarily through its Python API. As such, the
-    end-user tutorials and docs linked above focus on Python.
+Profiling and Optimization                          `Video <https://youtu.be/I4MjX598ZYs?list=PLGywud_-HlCORC0c4uj97oppQrGiB6JNy>`__                                     `Interactive Colab <https://colab.research.google.com/gist/eundersander/b62bb497519b44cf4ceb10e2079525dc/faster-rl-training-profiling-and-optimization.ipynb>`__
 
-    If you're looking for API reference of Habitat-sim's C++ internals, please see the
-    `C++ API <cpp.html>`_ tab.
+New Actions                                         :ref:`Web page <std:doc:new-actions>`
+
+Notebooks                                           :ref:`Web page <std:doc:notebooks>`
```

Comment:
This entry isn't really a tutorial. It just mentions a python notebook. Is this legacy? Should we rephrase or remove?

Reply:
Good catch, let's remove this one. I don't think the referenced notebook even exists anymore. Will need to be removed from the navbar pages lists in `conf.py` and `Doxyfile-mcss` also.